test/configCases/css/minimize-nesting-lowered/style.css

```
@@ -0,0 +1,69 @@
+	.h {
+		color: olive;
+	}
+}
+
+.i {
+	color: color-mix(in oklch, red, blue);
+}
+
+/* Kept: a declaration written after a nested rule is read after it. */
+.j {
+	color: red;
```

Stylelint (17.14.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)